packages/manager/apps/ips/src/utils/survey.constants.ts

```diff
@@ -0,0 +1 @@
+export const SURVEY_BASE_URL = 'https://survey.ovh.com/index.php/933956';
```

Comment:
Just put it directly in the SurveyLink.tsx file
There won't be any other constants related to this and other components won't need to access it